src/vector/platform/SeshatIndexManager.ts

```diff
@@ -0,0 +1,109 @@
+/*
+Copyright 2016 Aviral Dasgupta
+Copyright 2016 OpenMarket Ltd
+Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
+Copyright 2018 - 2021 New Vector Ltd
+Copyright 2022 Šimon Brandner <simon.bra.ag@gmail.com>
```

Comment:
I don't think this copyright is necessarily correct. At a minimum, we'd need to update the New Vector Ltd copyright, but I don't think refactoring maintains copyright.